model/Core/Classes/Relationship.md

```diff
@@ -11,6 +11,12 @@ Describes a relationship between one or more elements.
 A Relationship is a grouping of characteristics unique to an assertion
 that one Element is related to one or more other Elements in some way.
 
+To explicitly assert that no such relationships exist, the `to` property 
+should contain the 'NONE' individual.
+
+To explicitly assert that no assertions are being made regarding the 
+existence of such relationships, the `to` property should contain the 'NOASSERTION' individual.
```

Comment:
```suggestion
existence of such relationships, the `to` property should also contain the 'NOASSERTION' individual.
```

Reply:
This change does not look appropriate to me. Maybe I am misunderstanding its intent?
If the `to` property contains a value of NOASSERTION it should contain no other values similar to NONE.
If it does contain contain ANY other values then assertions are being made and NOASSERTION would not hold true.

I believe the clarification made for NONE ("and not other elements")should also be made here.

Reply:
Discssed with Max on call, and we'll not apply this suggestion from Max.

We'll go ahead and merge this as is now.